client/verta/verta/_internal_utils/_utils.py

```diff
@@ -342,6 +354,21 @@ def get_default_workspace(self):
         else:  # old backend
             return self.get_personal_workspace()
 
+    def get_custom_workspace(self, org_id, workspace_name):
```

Comment:
Could we move this function to `Client` and name it `_create_workspace()`?

Reply:
moved, renamed and refactored the method